js/curation-tool.js

```diff
@@ -892,23 +902,81 @@ const vm = new Vue({
+      // defaults.
+      if (!this.hasProperty(this.phylogenySpacingX, phylogenyIndex)) {
+        Vue.set(this.phylogenySpacingX, phylogenyIndex, DEFAULT_SPACING_X);
       }
-      if (!this.hasProperty(this.phylogenySpacingY, phylogeny)) {
-        Vue.set(this.phylogenySpacingY, phylogeny, 40);
+      if (!this.hasProperty(this.phylogenySpacingY, phylogenyIndex)) {
+        Vue.set(this.phylogenySpacingY, phylogenyIndex, DEFAULT_SPACING_Y);
       }
 
+      // Place and update the nodes on the phylogeny.
       tree
-        .spacing_x(this.phylogenySpacingX[phylogeny])
-        .spacing_y(this.phylogenySpacingY[phylogeny])
+        .spacing_x(this.phylogenySpacingX[phylogenyIndex])
+        .spacing_y(this.phylogenySpacingY[phylogenyIndex])
         .placenodes()
         .update();
       return tree;
     },
 
+    // Get and set phylogeny spacing information.
+    getPhylogenySpacingX(phylogenyIndex) {
+      // Return the phylogeny spacing in the X dimension for a particular phylogeny by index.
+      // Will set up the spacing variable if it has not been created yet, so this should be
+      // called before accessing this.phylogenySpacingX directly.
+      if (!this.hasProperty(this.phylogenySpacingX, phylogenyIndex)) {
+        Vue.set(this.phylogenySpacingX, phylogenyIndex, DEFAULT_SPACING_X);
+      }
+
+      return this.phylogenySpacingX[phylogenyIndex];
+    },
+
+    changePhylogenySpacingX(phylogenyIndex, changeBy) {
+      // Change the phylogeny spacing in the X dimension by the changeBy value.
+      // We prevent the value from going below MIN_SPACING_X.
+      if (!this.hasProperty(this.phylogenySpacingX, phylogenyIndex)) {
+        Vue.set(this.phylogenySpacingX, phylogenyIndex, DEFAULT_SPACING_X);
+      }
+
+      this.phylogenySpacingX[phylogenyIndex] += changeBy;
+
+      if (this.phylogenySpacingX[phylogenyIndex] < MIN_SPACING_X) {
+        this.phylogenySpacingX[phylogenyIndex] = MIN_SPACING_X;
+      }
+    },
+
+    getPhylogenySpacingY(phylogenyIndex) {
+      // Return the phylogeny spacing in the Y dimension for a particular phylogeny by index.
+      // Will set up the spacing variable if it has not been created yet, so this should be
+      // called before accessing this.phylogenySpacingY directly.
+      if (!this.hasProperty(this.phylogenySpacingY, phylogenyIndex)) {
+        Vue.set(this.phylogenySpacingY, phylogenyIndex, DEFAULT_SPACING_Y);
+      }
+
+      return this.phylogenySpacingY[phylogenyIndex];
+    },
+
+    changePhylogenySpacingY(phylogenyIndex, changeBy) {
+      // Change the phylogeny spacing in the Y dimension by the changeBy value.
+      // We prevent the value from going below MIN_SPACING_Y.
+      if (!this.hasProperty(this.phylogenySpacingY, phylogenyIndex)) {
+        Vue.set(this.phylogenySpacingY, phylogenyIndex, DEFAULT_SPACING_Y);
+      }
+
+      this.phylogenySpacingY[phylogenyIndex] += changeBy;
```

Comment:
Same as above (and would also need the following if condition to be deleted):
```suggestion
      let spacing = this.phylogenySpacingY[phylogenyIndex] + changeBy;
      this.phylogenySpacingY[phylogenyIndex] = spacing < MIN_SPACING_Y ? MIN_SPACING_Y : spacing;
```